lib/takeaway.rb

```diff
@@ -0,0 +1,53 @@
+class Takeaway
+  def initialize(menu, client)
+    @menu = menu
+    @client = client
+    @menu_array = @menu.menu
+    @current_selection = nil
+  end
+
+  attr_reader :current_selection
+
+  def list_food
+    puts "*** #{@menu.name} Menu! #{@menu.logo} ***"
+    @menu_array.each do |item|
+      puts "#{1 + @menu_array.index(item)}. #{item[:name]}. (£#{'%.2f' % item[:price]})"
+    end
+    puts "*** *** *** *** *** ***"
+  end
+
+  def order(selection)
```

Comment:
Instead of getting the information from the user via a specific input of a array of hashes you could consider using a method such as add_item(dish, quantity) to add items to the order. This would make your programme more user friendly. Also if you are expecting a typical quantity of 1 (as your readme suggests) you could consider setting a default value of quantity = 1 and then there would be no need for the user to specify the quantity unless they want more than 1.